pkgs/development/python-modules/ward/default.nix

```diff
@@ -36,7 +36,15 @@ buildPythonPackage rec {
     rich
     tomli
     pprintpp
-    cucumber-tag-expressions
+    (cucumber-tag-expressions.overridePythonAttrs (_: rec {
```

Comment:
We can't override version for modules. if possible the constraint of  `cucumber-tag-expressions` could be relaxed.

Otherwise we have to wait for upstream: https://github.com/darrenburns/ward/issues/380

Reply:
Understood, thanks for opening an issue upstream! Could I try changing the version constraints in the derivation?